### PR TITLE
Hclang 19 introduction

### PR DESCRIPTION
**Pull request by Abildgaard96**
### Description
Created Introduction for project.
Includes: 
    - Motivation for project
    - Quick intro to Arduino
    - Initial problem

### Changelog
 - Introduction added
 - Citations added to bibliography

### Issues solved 
add these with the respective key as HCLANG-1 (as an example)
 - HCLANG-19
 - HCLANG-21
 - HCLANG-27

## FOR REVIEWER [Do not remove]
A small checklist of things to note when you are doing a review. Make sure these things apply before going into master.
 - Did the PR actually fix the issue
 - Comment anything that doesn't make sense or anything you want clarified. Many comments are always ok
 - Make sure this is ready for exams if this is going into master
 - It's always a good idea having another reviewer look at a PR as well, if it's major. get a buddy.

### Regarding Report [if applicable]
- Does each line have a linechange in the sourcecode?
- Does sentences make sense?
- Did you do spellchecking?
- Citations?

### CODE REVIEW

File: report/1.Introduction/1.IntroductionCtrl.tex [hunk 1]
```diff
@@ -1,4 +1,33 @@
 
 \chapter{Introduction}
 
+In recent years, there has been a bigger focus on computer science, technology and programming in the danish school system.
+This is in large part due to the prediction that denmark will lack skilled programmers, cyber security specialists and other skilled computer science workers in the near future.
```
Comment (technight) [inline report/1.Introduction/1.IntroductionCtrl.tex]: DUDE! Denmark

File: report/1.Introduction/1.IntroductionCtrl.tex [hunk 2]
```diff
@@ -1,4 +1,33 @@
...
+teaches them the fundamentals of programming.
+
+One of the most common tools for introducing programming and electrical engineering in danish gymnasiums, is
+the Arduino development board. The Arduino platform is consistent of its own dedicated IDE and language.
+The Arduino programming language is based on C/C++ and designed to both be easy to use, as well as being
+more hardware-oriented.\cite{ArFAQ}
+The Arduino platform maintains its simplicity by oftentimes shielding away a lot of complexity by ways
+that are non-present in most programming languages.
+Although this is an advantage in terms of simplicity, it might halter, or even frustrate, the student when learning other programming
+languages.
```
Comment (technight) [inline report/1.Introduction/1.IntroductionCtrl.tex]: Why the line break?

File: report/1.Introduction/1.IntroductionCtrl.tex [hunk 3]
```diff
@@ -1,4 +1,33 @@
...
+
+As such the motivation for this project is to create a new language for the Arduino platform, with a focus on promoting simplicity and the thought process that goes into programming, rather than difficult syntax and extraneous setup code.
+
+%This project aims to create a new language for the Arduino platform that promotes simplicity and strives to teach student the fundamentals of programming.
+
+From this the following initial problem statement is drafted:
+
+\centering\textit{\textbf{How can a programming language for Arduino be optimized for learning without
```
Comment (technight) [inline report/1.Introduction/1.IntroductionCtrl.tex]: Centering should be stopped. This centers the rest of the document...

File: report/1.Introduction/1.IntroductionCtrl.tex [hunk 4]
```diff
@@ -1,4 +1,33 @@
...
+As a result, more students are being introduced to programming and programming languages during their duration at the
+danish gymnasiums. 
+The goal of this introduction is to give the students a basic idea of how the computers they use everyday function, but also to impart an interest or motivation for studying programming in their continued education.
+
+As programming languages are ever evolving and new languages are constantly designed it becomes relevant
+that students, who might be interested in studying programming, are introduced to the field in a way that
+teaches them the fundamentals of programming.
+
+One of the most common tools for introducing programming and electrical engineering in danish gymnasiums, is
+the Arduino development board. The Arduino platform is consistent of its own dedicated IDE and language.
```
Comment (jones1994) [inline report/1.Introduction/1.IntroductionCtrl.tex]: * change "consistent" to "consists of"

**Pull request merged**